docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx

```diff
@@ -1,32 +1,43 @@
-| <PyObject module="dagster" object="GraphDefinition" /> | A graph definition, which is a set of ops (or [sub-graphs](/concepts/ops-jobs-graphs/nesting-graphs)) wired together. Forms the core of a job. Typically constructed using the <PyObject object="job" decorator /> decorator. |
 
-_Jobs_ are the main unit of execution and monitoring in Dagster. The core of a job is a _graph_ of ops - connected via data dependencies. Jobs can also contain [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema), which determine some of the behavior of those ops.
 
-Ops are linked together by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that, in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies. This difference enables Dagster to support richer modeling of dependencies. Instead of merely ensuring that the order of execution is correct, dependencies in Dagster provide a variety of compile and run-time checks.
+Jobs are the main unit of execution and monitoring in Dagster. The core of a job is a [graph](/concepts/ops-jobs-graphs/graphs) of [ops](/concepts/ops-jobs-graphs/ops) connected via data dependencies.
 
-You can run a job in a variety of ways - in the Python process where it's defined, via the command line, from Dagit, or via a GraphQL API.
+Ops are linked together by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that, in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies.
 
-The [Dagit UI](/concepts/dagit/dagit) centers on jobs. Dagit makes it easy to view all the historical runs of each job in one place, as well as to manually kick off runs for a job.
+This difference enables Dagster to support richer modeling of dependencies. Instead of merely ensuring that the order of execution is correct, dependencies in Dagster provide a variety of compile and run-time checks.
 
-You can define [schedules](/concepts/partitions-schedules-sensors/schedules) to execute jobs at fixed intervals or define [sensors](/concepts/partitions-schedules-sensors/sensors) to trigger jobs when external changes occur.
+Using jobs, you can:
 
-## Creating a job
+- Determine op behavior using [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema)
+- Define [schedules](/concepts/partitions-schedules-sensors/schedules) to execute jobs at fixed intervals
+- Define [sensors](/concepts/partitions-schedules-sensors/sensors) to trigger jobs when external changes occur
+
+---
+
+## Creating jobs
+
+Jobs can be created in several ways:
```

Comment:
I wonder if it would be helpful to explain to the user when each of these ways is appropriate?

Reply:
It absolutely would. I'm not sure if I can get re-writing the content in as part of this, but I agree it is definitely something that should be included here.